frontend/src/pages/projects/notebook/NotebookImagePackageDetails.tsx

```diff
@@ -11,12 +11,12 @@ import { ImageVersionDependencyType } from '~/pages/projects/screens/spawner/typ
 
 type NotebookPackageDetailsProps = {
   dependencies: ImageVersionDependencyType[];
-  title?: string;
+  title?: React.ReactNode;
 };
 
 const NotebookImagePackageDetails: React.FC<NotebookPackageDetailsProps> = ({
   dependencies,
-  title,
+  title = <DescriptionListTerm>Packages</DescriptionListTerm>,
```

Comment:
You want to make title still default to just `Packages` and then keep this usage
```tsx
        <DescriptionListTerm>{title}</DescriptionListTerm>
```

reason is that if you put a custom title in that is just a string, you loose any formatting.